src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,144 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K,V>[] table = new Node[DEFAULT_INITIAL_CAPACITY];
+    private int capacity;
+    private int size;
+    private int threshold;
+
+    class Node<K,V> {
```

Comment:
private 

Private ones (methods, classes) should be after public ones.

Reply:
not fixed

Reply:
Should I move Node class to the bottom?
I don't fully understand the correct order

Reply:
If you have any questions - do not ask in the PR
Welcome to chat - it is designed for discussions and additional explanations

Reply:
yes, move Node class to the bottom